Characters/Projectiles/Projectile.cs

```diff
@@ -4,17 +4,29 @@
 using Physics.Collisions;
 
 namespace Character.Projectiles;
+/// <summary>
+/// A projectile that can be fired by a character.
+/// </summary>
 public class Projectile : ISimulationMember
 {
-    public BodyHandle BodyHandle { get; private set; }
-
+    /// <summary>
+    /// Whether or not the projectile is dead and should be removed from the scene.
+    /// </summary>
     public bool IsDead { get; private set; }
-
-    public ProjectileMesh Mesh { get; private set; }
-
+    /// <summary>
```

Comment:
Maybe empty line between public bool IsDead { get; private set; } and the docs for the next property?